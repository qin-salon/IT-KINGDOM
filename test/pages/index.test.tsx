import IndexPage from "src/pages";
import { render } from "test/test-utils";

describe("Index page", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<IndexPage subscriberCount={70000} />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
