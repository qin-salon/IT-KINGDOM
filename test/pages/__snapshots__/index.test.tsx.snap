// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="container"
  >
    <h1>
      <a
        href="/"
      >
        <picture>
          <source
            media="(max-width: 320px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <source
            media="(min-width: 321px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <source
            media="(max-width: 320px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <source
            media="(min-width: 321px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <img
            alt="Logo of 'IT KINGDOM"
            src="test-file-stub"
          />
        </picture>
      </a>
    </h1>
    <nav
      class="sansSerif nav"
    >
      <ul>
        <li>
          <a
            href="/faq"
          >
            よくある質問
          </a>
        </li>
        <li>
          <a
            href="https://qin.salon"
            rel="noopener noreferrer"
            target="_blank"
          >
            入国手続
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <main
    class="container"
  >
    <section
      class="jsx-3651463820 container"
    >
      <picture
        class="jsx-3651463820 img"
      >
        <source
          class="jsx-3651463820"
          media="(max-width:768px)"
          srcset=" img/hero/sp-400.webp 400w, img/hero/sp-500.webp 500w, img/hero/sp-600.webp 600w, img/hero/sp-768.webp 768w, img/hero/sp.webp"
          type="image/webp"
        />
        <source
          class="jsx-3651463820"
          srcset=" img/hero/pc-980.webp 980w, img/hero/pc-1200.webp 1200w, img/hero/pc-1680.webp 1680w, img/hero/pc-2000.webp 2000w, img/hero/pc-2560.webp 2560w, img/hero/pc.webp"
          type="image/webp"
        />
        <source
          class="jsx-3651463820"
          media="(max-width:768px)"
          srcset=" img/hero/sp-400.png 400w, img/hero/sp-500.png 500w, img/hero/sp-600.png 600w, img/hero/sp-768.png 768w, img/hero/sp.png"
          type="image/png"
        />
        <source
          class="jsx-3651463820"
          srcset=" img/hero/pc-980.png 980w, img/hero/pc-1200.png 1200w, img/hero/pc-1680.png 1680w, img/hero/pc-2000.png 2000w, img/hero/pc-2560.png 2560w, img/hero/pc.png"
          type="image/png"
        />
        <img
          alt="King shimabu and soldiers"
          class="jsx-3651463820"
          src="img/hero/pc.png"
        />
      </picture>
      <div
        class="jsx-3651463820"
      >
        <h1
          class="jsx-3651463820 text"
        >
          <div
            class="jsx-3651463820 line"
          >
            <span
              class="jsx-3651463820"
            >
              チャンネル登録者
            </span>
            <span
              class="jsx-3651463820 number number"
            >
              7万人
            </span>
            <span
              class="jsx-3651463820"
            >
              の
            </span>
          </div>
          <div
            class="jsx-3651463820 line"
          >
            <span
              class="jsx-3651463820"
            >
              エンジニア
            </span>
            <span
              class="jsx-3651463820 strong"
            >
              YouTuber
            </span>
            <span
              class="jsx-3651463820"
            >
              がおくる
            </span>
          </div>
          <div
            class="jsx-3651463820 line"
          >
            <span
              class="jsx-3651463820 strong"
            >
              オンラインサロン
            </span>
          </div>
        </h1>
        <div
          class="jsx-3651463820 itk_button"
        >
          <picture>
            <source
              media="(max-width: 400px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 401px) and (max-width: 680px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 681px) and (max-width: 800px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 801px) and (max-width: 1050px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 1051px) and (max-width: 1480px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 1481px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(max-width: 400px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 401px) and (max-width: 680px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 681px) and (max-width: 800px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 801px) and (max-width: 1050px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 1051px) and (max-width: 1480px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 1481px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <img
              alt="Logo of 'IT KINGDOM"
              class="itk"
              src="test-file-stub"
            />
          </picture>
          <div
            class="jsx-3651463820 button"
          >
            <a
              class="jsx-3651463820"
              href="https://qin.salon"
              rel="noopener noreferrer"
              target="_blank"
            >
              <picture>
                <source
                  media="(max-width: 320px)"
                  srcset=", undefined 2x"
                  type="image/undefined"
                />
                <source
                  media="(min-width: 321px)"
                  srcset=", undefined 2x"
                  type="image/undefined"
                />
                <source
                  media="(max-width: 320px)"
                  srcset=", undefined 2x"
                  type="image/undefined"
                />
                <source
                  media="(min-width: 321px)"
                  srcset=", undefined 2x"
                  type="image/undefined"
                />
                <img
                  alt="Entry button"
                  src="test-file-stub"
                />
              </picture>
            </a>
          </div>
        </div>
      </div>
    </section>
    <section
      class="container"
    >
      <h1>
        CONCEPT VIDEO
      </h1>
      <div
        class="responsiveVideo"
      >
        <iframe
          allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
          allowfullscreen=""
          frameborder="0"
          height="405"
          loading="lazy"
          title="Concept video"
          width="720"
        />
      </div>
      <picture>
        <source
          media="(max-width: 520px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 521px) and (max-width: 768px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 769px) and (max-width: 980px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 981px) and (max-width: 1200px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 1201px) and (max-width: 1680px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 1681px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(max-width: 520px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 521px) and (max-width: 768px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 769px) and (max-width: 980px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 981px) and (max-width: 1200px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 1201px) and (max-width: 1680px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 1681px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <img
          alt="Background smoke"
          class="smoke"
          loading="lazy"
          src="test-file-stub"
        />
      </picture>
    </section>
    <section
      class="container"
    >
      <picture>
        <source
          media="(max-width: 520px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 521px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(max-width: 520px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 521px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <img
          alt="Vision section's icon"
          class="icon front"
          loading="lazy"
          src="test-file-stub"
        />
      </picture>
      <h1
        class="title front"
      >
        <strong>
          最高
        </strong>
        の
        <span>
          プロダクト
        </span>
        を生み出す
        <br />
        <strong>
          最強
        </strong>
        の
        <span>
          コミュニティ
        </span>
        をつくりたい
      </h1>
      <p
        class="sansSerif paragraph front"
      >
        私はこのサロンから世界中で使われる最高のプロダクトを生み出したいと思っています。
みんなでスキルアップして良いプロダクトを作りましょう。
      </p>
      <div
        class="frame"
      >
        <h2>
          そのために
          <strong>
            私
          </strong>
          がやること
        </h2>
        <p
          class="sansSerif"
        >
          プロダクトを作るために必要なプログラミングスキルを伝授します。
          <br />
          必要に応じてデザイン・マーケティング・プレゼンテーション・経営企画などの講座も行います。
        </p>
        <picture>
          <source
            media="(max-width: 9999px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <source
            media="(max-width: 9999px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <img
            alt="Background"
            class="goldTopLeft"
            loading="lazy"
            src="test-file-stub"
          />
        </picture>
        <picture>
          <source
            media="(max-width: 9999px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <source
            media="(max-width: 9999px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <img
            alt="Shimabu"
            class="shimabu_udegumi"
            loading="lazy"
            src="test-file-stub"
          />
        </picture>
      </div>
      <div
        class="frame"
      >
        <h2>
          <strong>
            みんな
          </strong>
          にやってもらうこと
        </h2>
        <p
          class="sansSerif"
        >
          最高のプロダクトが作れるように、各自スキルアップに励む必要があります。
          <br />
          学んだことは積極的にアウトプットを行い、コミュニティ全体を強くしていきましょう。
        </p>
        <picture>
          <source
            media="(max-width: 9999px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <source
            media="(max-width: 9999px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <img
            alt="Background"
            class="goldBottomRight"
            loading="lazy"
            src="test-file-stub"
          />
        </picture>
        <picture>
          <source
            media="(max-width: 9999px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <source
            media="(max-width: 9999px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <img
            alt="Background"
            class="goldTopLeft"
            loading="lazy"
            src="test-file-stub"
          />
        </picture>
        <picture>
          <source
            media="(max-width: 9999px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <source
            media="(max-width: 9999px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <img
            alt="Salon members"
            class="soldiers"
            loading="lazy"
            src="test-file-stub"
          />
        </picture>
      </div>
      <small
        class="sansSerif note"
      >
        ※ プロジェクトの動きや中の様子を見たいだけの方もご参加いただけます。
      </small>
      <span
        class="backgroundText sai"
      >
        最
      </span>
      <span
        class="backgroundText kou"
      >
        高
      </span>
      <span
        class="backgroundText kyou"
      >
        強
      </span>
    </section>
    <section
      class="container"
    >
      <h1>
        CONTENTS
      </h1>
      <ul>
        <li
          class="content"
          style="background-image: url(/img/ink.png);"
        >
          <picture>
            <source
              media="(max-width: 400px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 401px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(max-width: 400px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 401px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <img
              alt="プロダクト開発"
              loading="lazy"
              src="test-file-stub"
            />
          </picture>
          <h2>
            <span>
              01. 
            </span>
            プロダクト開発
          </h2>
          <p
            class="sansSerif"
          >
            私含め現役エンジニアが新規事業のプロジェクトをたてます。プロジェクトのコードはみんなが読めるようにGitHubを使用し、適宜コードの解説も行います。開発に参加したい方は、実際にコードを書いてチーム開発の実戦を経験していただくことも可能です。
          </p>
        </li>
        <li
          class="content"
        >
          <picture>
            <source
              media="(max-width: 400px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 401px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(max-width: 400px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 401px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <img
              alt="気軽に質問"
              loading="lazy"
              src="test-file-stub"
            />
          </picture>
          <h2>
            <span>
              02. 
            </span>
            気軽に質問
          </h2>
          <p
            class="sansSerif"
          >
            Qastという外部サービスを使用します。Qastには匿名で質問を行う機能があり「何度も質問するのは気が引ける」「初歩的すぎて聞くのが恥ずかしい」という方も気軽に質問できます。また誰でも回答でき、回答していくことでポイントが溜まる仕組みがあります。楽しく教え合いましょう。
          </p>
        </li>
        <li
          class="content"
          style="background-image: url(/img/ink.png);"
        >
          <picture>
            <source
              media="(max-width: 400px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 401px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(max-width: 400px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 401px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <img
              alt="ブログ & アウトプット"
              loading="lazy"
              src="test-file-stub"
            />
          </picture>
          <h2>
            <span>
              03. 
            </span>
            ブログ & アウトプット
          </h2>
          <p
            class="sansSerif"
          >
            Qastにはブログ機能も備わっており、誰でもブログを書くことができます。外部だと心無い指摘が少し怖いかもしれませんが、当サロンではみんなが優しく教えてくれますので、積極的にアウトプットしていきましょう。また私含め現役エンジニアもブログを執筆します。
          </p>
        </li>
        <li
          class="content"
        >
          <picture>
            <source
              media="(max-width: 400px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 401px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(max-width: 400px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 401px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <img
              alt="IT業界研究"
              loading="lazy"
              src="test-file-stub"
            />
          </picture>
          <h2>
            <span>
              04. 
            </span>
            IT業界研究
          </h2>
          <p
            class="sansSerif"
          >
            優れたプロダクトを作るためにはIT業界のトレンドを知る必要があります。私がニュースをサロン内でシェアしたり時事ネタを解説するので、みんなで議論しましょう。また就職や転職に必要なIT企業の研究や情報共有も行います。
          </p>
        </li>
        <li
          class="content"
          style="background-image: url(/img/ink.png);"
        >
          <picture>
            <source
              media="(max-width: 400px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 401px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(max-width: 400px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 401px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <img
              alt="楽しめるコミュニティ"
              loading="lazy"
              src="test-file-stub"
            />
          </picture>
          <h2>
            <span>
              05. 
            </span>
            楽しめるコミュニティ
          </h2>
          <p
            class="sansSerif"
          >
            ただ学ぶだけではなく楽しめるサロンにします。Microsoft Teamsを用いて頻繁に交流の場を設けます。一緒に勉強するもくもく会、質疑応答をライブ配信、趣味のチャットグループ、オンライン飲み会、オフラインでの交流会などを予定しています。
          </p>
          <ul>
            <li
              class="sansSerif"
            >
              ※ オフラインイベントは新型コロナウイルス感染症の影響が落ち着いてから計画いたします。
            </li>
          </ul>
        </li>
      </ul>
      <div
        class="more"
      >
        <h2
          class="headline container coloredTitle"
        >
          <span>
            AND MORE
          </span>
          <span>
            他にも企画中
          </span>
        </h2>
        <p
          class="sansSerif"
        >
          当サロンは最高のプロダクトを生み出すために常に進化していく必要があります。
          サロンのあり方を一緒に考えみんなで成長させていくことが重要です。
          ここに書いてあること以外にもおもしろいことは積極的にやりましょう。
        </p>
      </div>
    </section>
    <section
      class="container"
    >
      <div
        class="title_step"
      >
        <h1
          class="title"
        >
          <span>
            訓練
          </span>
          <span>
            STEP UP
          </span>
        </h1>
        <picture>
          <source
            media="(max-width: 400px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <source
            media="(min-width: 401px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <source
            media="(max-width: 400px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <source
            media="(min-width: 401px)"
            srcset=", undefined 2x"
            type="image/undefined"
          />
          <img
            alt="Curriculum"
            class="step"
            loading="lazy"
            src="test-file-stub"
          />
        </picture>
      </div>
      <ul
        class="text"
      >
        <li>
          <h2>
            Lv.1  隊員〜什長
          </h2>
          <p
            class="sansSerif"
          >
            まずはプログラミングの基礎知識を身につけよう。気軽に質問できる環境に加え、ライブ講座や簡単な課題も用意します。
          </p>
        </li>
        <li>
          <h2>
            Lv.2  百人将〜千人将
          </h2>
          <p
            class="sansSerif"
          >
            実際に開発に参加しよう。適宜コード解説も行い、まずはコードを読めるようにします。その後は実際にコードを書いてみましょう。
          </p>
        </li>
        <li>
          <h2>
            Lv.3  二千人将〜五千人将
          </h2>
          <p
            class="sansSerif"
          >
            プロジェクトをリードしていこう。新機能追加、コードレビュー、リファクタリングなどプロダクトを改善させてみましょう。
          </p>
        </li>
        <li>
          <h2>
            Lv.4  将軍、大将軍
          </h2>
          <p
            class="sansSerif"
          >
            ここまできたら一人前。1つのプロジェクトだけでなくコミュニティ全体をリードできるように、みんなを引っ張っていってください！
          </p>
        </li>
      </ul>
    </section>
    <section
      class="container"
    >
      <h1
        class="headline container"
      >
        <span>
          MEMBERS
        </span>
        <span>
          運営
        </span>
      </h1>
      <ul
        class="members"
      >
        <li>
          <picture>
            <source
              media="(max-width: 480px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 481px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(max-width: 480px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 481px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <img
              alt="しまぶー"
              loading="lazy"
              src="test-file-stub"
            />
          </picture>
          <h1>
            しまぶー
          </h1>
          <h2>
            国王
          </h2>
          <p
            class="sansSerif"
          >
            ITエンジニアYouTuber。神戸大学経営学部卒。未経験から独学でプログラミングを勉強し、新卒でヤフーに入社。2019年に株式会社GameHintを創業。
          </p>
        </li>
        <li>
          <picture>
            <source
              media="(max-width: 480px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 481px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(max-width: 480px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 481px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <img
              alt="まっつん"
              loading="lazy"
              src="test-file-stub"
            />
          </picture>
          <h1>
            まっつん
          </h1>
          <h2>
            相国
          </h2>
          <p
            class="sansSerif"
          >
            神戸大学工学部卒。新卒でヤフーに入社し、決済部署でインフラ・バックエンドを担当。現在はしまぶーと一緒に起業。サロンではバックエンドを見る。
          </p>
        </li>
        <li>
          <picture>
            <source
              media="(max-width: 480px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 481px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(max-width: 480px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 481px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <img
              alt="いっぽ"
              loading="lazy"
              src="test-file-stub"
            />
          </picture>
          <h1>
            いっぽ
          </h1>
          <h2>
            左丞相
          </h2>
          <p
            class="sansSerif"
          >
            神戸大学工学部卒。新卒でクックパッドに入社し現在はITベンチャーに転職。アプリ・フロントエンド・デザインに秀でており、サロンでもその分野を担当。
          </p>
        </li>
        <li>
          <picture>
            <source
              media="(max-width: 480px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 481px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(max-width: 480px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 481px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <img
              alt="やっさん"
              loading="lazy"
              src="test-file-stub"
            />
          </picture>
          <h1>
            やっさん
          </h1>
          <h2>
            右丞相
          </h2>
          <p
            class="sansSerif"
          >
            神戸大学工学部卒。某有名企業の現役エンジニア。幅広いエンジニアリングの知識を持っている。サロンでは分野を問わず幅広く担当する。
          </p>
        </li>
      </ul>
    </section>
    <div
      class="container"
    >
      <picture>
        <source
          media="(max-width: 520px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 521px) and (max-width: 768px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 769px) and (max-width: 980px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 981px) and (max-width: 1200px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 1201px) and (max-width: 1680px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 1681px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(max-width: 520px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 521px) and (max-width: 768px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 769px) and (max-width: 980px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 981px) and (max-width: 1200px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 1201px) and (max-width: 1680px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <source
          media="(min-width: 1681px)"
          srcset=", undefined 2x"
          type="image/undefined"
        />
        <img
          alt="Background"
          class="img"
          loading="lazy"
          src="test-file-stub"
        />
      </picture>
      <section
        class="container"
      >
        <div
          class="backgroundText"
        >
          <h1
            class="headline container coloredTitle"
          >
            <span>
              MESSAGE
            </span>
            <span>
              国王のメッセージ
            </span>
          </h1>
          <div
            class="text"
          >
            <p
              class="sansSerif"
            >
              IT KINGDOMはただプログラミングを教えるだけのサロンではありません。
              学んで終わりではなくプロダクトをリリースし、実際にビジネスとして展開していきます。
              そこで得た収益をサロンに還元する、それが私が目指す世界です。
            </p>
            <p
              class="sansSerif"
            >
              私はその実現に向けてサロンメンバーのIT力向上に注力します。
              ITエンジニアを目指す方も多いと思われますが、サロンではそのサポートもいたします。
              みなさんがIT企業に就職し、更にスキルアップして、また一緒に開発できることを願っています。
            </p>
            <p
              class="sansSerif"
            >
              このような形で実際にビジネスを行うのは前例の殆どない最先端な取り組みです。
              私も不安ではありますが、そういった不安も一緒に楽しめる方に入っていただけると嬉しいです。
              個々の進化はもちろん、コミュニティの進化も目指して、一緒に最高のモノづくりをしましょう。
            </p>
          </div>
        </div>
      </section>
      <section
        class="container"
      >
        <h1
          class="headline container"
        >
          <span>
            PRICE
          </span>
          <span>
            料金
          </span>
        </h1>
        <div>
          <div
            class="price"
          >
            <span
              class="sansSerif"
            >
              月額 (税抜)
            </span>
            <h2>
              2,980
              <span>
                円
              </span>
            </h2>
          </div>
          <p
            class="sansSerif note"
          >
            Qastなどの外部サービス利用料は含まれています。
            オフラインのイベントは別途費用がかかります。
          </p>
        </div>
      </section>
      <section
        class="container"
      >
        <h1
          class="headline container"
        >
          <span>
            HOW TO ENTRY
          </span>
          <span>
            入国方法
          </span>
        </h1>
        <ul
          style="background-image: url(img/ink.png);"
        >
          <li
            class="step"
          >
            <div>
              <span>
                STEP
              </span>
              <span>
                1
              </span>
            </div>
            <h2>
              入国手続きを行う
            </h2>
            <p
              class="sansSerif"
            >
              サロンに参加するにはページ上部または下部にある「入国手続きを行う」ボタンから行えます。GitHubまたはGoogleのアカウント連携で登録が可能です。プログラミングを学びたい方にはGitHub連携をオススメしています。
            </p>
          </li>
          <li
            class="step"
          >
            <div>
              <span>
                STEP
              </span>
              <span>
                2
              </span>
            </div>
            <h2>
              月会費をお支払いする
            </h2>
            <p
              class="sansSerif"
            >
              アカウント登録後に会費をお支払いする画面（Stripe）に遷移します。お支払い手続きをお済ませください。
            </p>
          </li>
          <li
            class="step"
          >
            <div>
              <span>
                STEP
              </span>
              <span>
                3
              </span>
            </div>
            <h2>
              プロフィールを登録する
            </h2>
            <p
              class="sansSerif"
            >
              続いてプロフィールをご記入ください。イベント企画の際に、こちらでご記入いただいたプロフィール情報を参考にさせていただきます。後からでも変更できます。
            </p>
          </li>
          <li
            class="step"
          >
            <div>
              <span>
                STEP
              </span>
              <span>
                4
              </span>
            </div>
            <h2>
              各種サービスへご招待
            </h2>
            <p
              class="sansSerif"
            >
              当サロンでは「Qast」「Microsoft Teams」という2つのサービスを使用します。プロフィールを登録後、1日後を目安にメールアドレスへ招待が届きますので、そこからご登録ください。使い方はサロン内で解説いたします。
            </p>
          </li>
          <li
            class="step"
          >
            <div>
              <span>
                STEP
              </span>
              <span>
                5
              </span>
            </div>
            <h2>
              学ぼう、育てよう、楽しもう
            </h2>
            <p
              class="sansSerif"
            >
              ご自身の目標実現のため、スキルアップに励むも良し！ときにはコミュニティの成長のため、サロンに尽くすも良し！さぁ、楽しい IT KINGDOM ライフの始まりです！
            </p>
          </li>
        </ul>
        <a
          class="button"
          href="https://qin.salon"
          rel="noopener noreferrer"
          target="_blank"
        >
          <picture>
            <source
              media="(max-width: 320px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 321px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(max-width: 320px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <source
              media="(min-width: 321px)"
              srcset=", undefined 2x"
              type="image/undefined"
            />
            <img
              alt="Entry button"
              loading="lazy"
              src="test-file-stub"
            />
          </picture>
        </a>
      </section>
    </div>
  </main>
  <footer
    class="sansSerif container"
  >
    <ul
      class="links"
    >
      <li
        class="link"
      >
        <a
          href="/support/terms"
        >
          利用規約
        </a>
      </li>
      <li
        class="link"
      >
        <a
          href="/support/privacy-policy"
        >
          プライバシーポリシー
        </a>
      </li>
      <li
        class="link"
      >
        <a
          href="/support/commerce"
        >
          特定商取引法に基づく表記
        </a>
      </li>
      <li
        class="link"
      >
        <a
          href="https://docs.google.com/forms/d/e/1FAIpQLSf7lYhGHmnl2KTPXsArv5KCoZvlwX7pMF7EZrDXSgWQWnJDhw/viewform"
          rel="noopener noreferrer"
          target="_blank"
        >
          お問い合わせ
        </a>
      </li>
    </ul>
    <small
      class="copyright"
      lang="en"
    >
      @ 2020 IT KINGDOM
    </small>
  </footer>
</DocumentFragment>
`;
